### PR TITLE
SE-3090: remove `valuation/risk/spot delta` from equity options notebook

### PR DESCRIPTION
**Pull request by daniel-ilyin**
# Pull Request Checklist

- [x] Changes follow the [style guide](https://github.com/finbourne/sample-notebooks/blob/master/docs/FINBOURNE%20notebook%20style%20guide.ipynb)
- [] Tests pass
- [x] Raised the PR against the `develop` branch

# Description of the PR

Describe the code changes for the reviewers, explain the solution you have provided and how it fixes the issue

### CODE REVIEW

File: Index.ipynb
Comment (dgbarclay) [inline Index.ipynb:1]: do we need to change this file?

Comment (daniel-ilyin) [reply Index.ipynb:1]: Reverted change, accidentally shelved it

Comment (dgbarclay) [reply Index.ipynb:1]: thanks

**Review by jyaffe — APPROVED** [review]

**Pull request merged**